Revoke-Obfuscation.psm1

```diff
@@ -160,6 +168,14 @@ http://www.leeholmes.com/blog/
         [Switch]
         $CommandLine,
 
+        [Parameter(Mandatory = $false)]
+        [Switch]
+        $NormalizedFeatures,
```

Comment:
Yes, it is possible to specify both -NormalizedFeatures and -WebScan, but it will only run WebScan anyway.
I changed the names of the parameters to 'Normalized' and 'AzureML', but this interface is used in some functions that I haven't even looked at before. I can work on it when I'm done with my other tasks.